configure.ac

```diff
@@ -933,7 +935,6 @@ then
     test "$enable_psk" = "" && enable_psk=yes
     test "$enable_cmac" = "" && enable_cmac=yes
     test "$enable_siphash" = "" && enable_siphash=yes
-    test "$enable_aesxts" = "" && enable_aesxts=yes
```

Comment:
Why remove AES-XTS from 'all-crypt'?
It looks like it is still in 'all'.
Is it because it is now on for non-FIPS?
This looks inconsistent to me.

Reply:
it's not removed from all-crypto -- all and all-crypto have the same clause enabling it `if test "$ENABLED_FIPS" = "no" || test "$FIPS_VERSION" = "dev"`.

XTS isn't part of fips <5.3, and "dev" currently corresponds to 5.3.  when we allocate a formal version for the next submission these clauses will get updated accordingly.